### PR TITLE
Fixing CUP reb factions

### PR DESCRIPTION
**Pull request by killerswin2**
## What type of PR is this.
1. [ ] Bug
2. [x] Change
3. [ ] Enhancement

### What have you changed and why?
Information:
    fixed the missing flag texture in NPC, fixed incorrect flag texture in TKL, removed towing tractor in the civ vehicles. Removed confusing starting weapons, switched starting weapons for TKL to be enfields. Removed headgear as it gets auto auto dectected in anyways. Removed uniforms that may causes issues. Made NPC uniforms more temperate, made TKL uniforms more arid.

### Please verify the following and ensure all checks are completed.

1. [x] Have you loaded the mission in LAN host?
2. [ ] Have you loaded the mission on a dedicated server?

### Is further testing or are further changes required?
1. [x] No
2. [ ] Yes (Please provide further detail below.)

### How can the changes be tested?
Steps: 

********************************************************
Notes:

### CODE REVIEW

**Comment by killerswin2** [comment]
not now, waiting for testing feedback

**Comment by killerswin2** [comment]
Laze, I'm just gonna change Reb stuff in this pr.

File: A3A/addons/core/Templates/Templates/CUP/CUP_Reb_NPC_Temperate.sqf [hunk 1]
```diff
@@ -5,7 +5,7 @@
 ["name", "NPC"] call _fnc_saveToTemplate;                         //this line determines the faction name -- Example: ["name", "NATO"] - ENTER ONLY ONE OPTION
```
Comment (Lazejun) [inline A3A/addons/core/Templates/Templates/CUP/CUP_Reb_NPC_Temperate.sqf:5]: "NPC" should be called Napa as it is Napa, by Uniforms, by Flags, by most configs, by Arma Lore.
Napa is used on all Maps exept Takistan and Kunduz.

File: A3A/addons/core/Templates/Templates/CUP/CUP_Reb_NPC_Temperate.sqf [hunk 2]
```diff
@@ -43,14 +43,14 @@
...
-    "CUP_sgun_CZ584", "CUP_srifle_CZ550_rail", "CUP_srifle_Remington700", "CUP_sgun_slamfire",
-    "CUP_1Rnd_12Gauge_Pellets_No00_Buck", "CUP_1Rnd_12Gauge_Pellets_No3_Buck", "CUP_5x_22_LR_17_HMR_M", "CUP_6Rnd_762x51_R700", "CUP_1Rnd_762x51_CZ584",
+    "CUP_srifle_Remington700", "CUP_sgun_slamfire",
+    "CUP_1Rnd_12Gauge_Pellets_No00_Buck", "CUP_1Rnd_12Gauge_Pellets_No3_Buck", "CUP_6Rnd_762x51_R700",
     "CUP_hgun_M9", "CUP_hgun_TaurusTracker455", "CUP_15Rnd_9x19_M9", "CUP_6Rnd_45ACP_M",
     "CUP_launch_RPG18",
     "CUP_HandGrenade_RGD5", "SmokeShell",
     "CUP_V_I_Carrier_Belt", "CUP_V_I_Guerilla_Jacket", "CUP_V_I_RACS_Carrier_Rig_2", "CUP_V_I_RACS_Carrier_Rig_wdl_2",
     "CUP_V_RUS_Smersh_New_Light", "CUP_V_OI_TKI_Jacket1_06", "CUP_V_OI_TKI_Jacket5_05", "CUP_V_OI_TKI_Jacket5_06", "CUP_V_OI_TKI_Jacket3_04",
-    "B_FieldPack_cbr", "B_FieldPack_khk", "B_FieldPack_oli","B_AssaultPack_cbr","B_AssaultPack_rgr","B_AssaultPack_khk",
```
Comment (Lazejun) [inline A3A/addons/core/Templates/Templates/CUP/CUP_Reb_NPC_Temperate.sqf:53]: Why are you reducing all Backpacks to the Coyte Assaultpack and the Fieldpack?

File: A3A/addons/core/Templates/Templates/CUP/CUP_Reb_TKL_Arid.sqf [hunk 3]
```diff
@@ -5,7 +5,7 @@
 ["name", "TKL"] call _fnc_saveToTemplate;                         //this line determines the faction name -- Example: ["name", "NATO"] - ENTER ONLY ONE OPTION
```
Comment (Lazejun) [inline A3A/addons/core/Templates/Templates/CUP/CUP_Reb_TKL_Arid.sqf:5]: If you see no point of actually making this a Rebell Faction then just get rid of it and use Napa for everything.

File: A3A/addons/core/Templates/Templates/CUP/CUP_Reb.sqf [hunk 4]
```diff
@@ -2,10 +2,10 @@
 //   Rebel Information   //
 ///////////////////////////
 
-["name", "NPC"] call _fnc_saveToTemplate;                         //this line determines the faction name -- Example: ["name", "NATO"] - ENTER ONLY ONE OPTION
+["name", "NAPA"] call _fnc_saveToTemplate;                         //this line determines the faction name -- Example: ["name", "NATO"] - ENTER ONLY ONE OPTION
 
 ["flag", "Flag_NATO_F"] call _fnc_saveToTemplate;                         //this line determines the flag -- Example: ["flag", "Flag_NATO_F"] - ENTER ONLY ONE OPTION
-["flagTexture", ""] call _fnc_saveToTemplate;                 //this line determines the flag texture -- Example: ["flagTexture", "\A3\Data_F\Flags\Flag_NATO_CO.paa"] - ENTER ONLY ONE OPTION
+["flagTexture", "\CUP\BaseConfigs\CUP_BaseConfigs\data\Flags\flag_napa_co.paa"] call _fnc_saveToTemplate;                 //this line determines the flag texture -- Example: ["flagTexture", "\A3\Data_F\Flags\Flag_NATO_CO.paa"] - ENTER ONLY ONE OPTION
```
Comment (Lazejun) [inline A3A/addons/core/Templates/Templates/CUP/CUP_Reb.sqf:8]: "cup\baseconfigs\cup_baseconfigs\data\flags\flag_napa_co.paa"

File: A3A/addons/core/Templates/Templates/CUP/CUP_Reb.sqf [hunk 5]
```diff
@@ -55,16 +55,16 @@ private _initialRebelEquipment = [
 ];
 
 if (A3A_hasTFAR) then {_initialRebelEquipment append ["tf_microdagr", "tf_anprc154"]};
-if (A3A_hasTFAR && startWithLongRangeRadio) then {_initialRebelEquipment pushBack "tf_anprc155"};
+if (A3A_hasTFAR && startWithLongRangeRadio) then {_initialRebelEquipment pushBack "TFAR_anprc155_coyote"};
```
Comment (Lazejun) [inline A3A/addons/core/Templates/Templates/CUP/CUP_Reb.sqf:58]: Give both, they are the same just different colors, that will also remove it from loot so you dont unlock the same variant.

**Review by Lazejun — APPROVED** [review]

**Pull request merged**